pkg/metrics/metrics.go

```diff
@@ -93,12 +93,17 @@ var (
 	// total number of work object processing requests.
 	//
 	// The following labels are available:
-	// * apply_status: the apply status of the processing request; values can
-	//   be "applied", "failed", or "skipped".
+	// * apply_status: the apply status of the processing request; see the list of
+	//   work apply condition reasons in the work applier source code for possible values.
```

Comment:
Could we have a link or file pointer here?

Reply:
Done.